cli/flags.rs

```diff
@@ -1375,6 +1376,14 @@ fn test_subcommand<'a>() -> App<'a> {
         .help("Ignore files")
         .value_hint(ValueHint::AnyPath),
     )
+    .arg(
+      Arg::new("update")
+        .long("update")
+        .short('u')
+        .help("Enable snapshot update mode")
+        .takes_value(false)
+        .required(false),
+    )
```

Comment:
I'm against adding this flag right now; it's unclear ATM how to snapshotting functionality will work. We should first have an MVP in `deno_std` before we start adding new flags to `deno test`